### PR TITLE
Test build fix

### PR DESCRIPTION
**Pull request by nickitat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit aa117869d673ed5490a36580d1e6b4836becf40f with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/71988/aa117869d673ed5490a36580d1e6b4836becf40f/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/f29d7841fe01ccfdb28e9469ec60b93db99ed7d5/docs_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: CMakeLists.txt [hunk 1]
```diff
@@ -64,8 +64,8 @@ if (ENABLE_CHECK_HEAVY_BUILDS)
     # Sanitizers are too heavy. Some architectures too.
     if (SANITIZE OR SANITIZE_COVERAGE OR WITH_COVERAGE OR ARCH_RISCV64 OR ARCH_LOONGARCH64)
         # Twice as large
-        set (RLIMIT_DATA 10000000000)
```
Comment (alexey-milovidov) [inline CMakeLists.txt:67]: ?

Comment (nickitat) [reply CMakeLists.txt:67]: build fails on loongarch with 

```
2024-11-15T11:58:29.6461264Z Nov 15 12:58:29 sccache: warning: The server looks like it shut down unexpectedly, compiling locally instead
2024-11-15T11:58:29.6461953Z Nov 15 12:58:29 sccache: Compile terminated by signal 9
```

i suspect that it ooms

Comment (Algunenano) [reply CMakeLists.txt:67]: It's there precisely to prevent us from adding more complex files that explode the compilation times and memory usage

Comment (alexey-milovidov) [reply CMakeLists.txt:67]: Fixed in https://github.com/ClickHouse/ClickHouse/pull/71945

Please remove this modification.

**Pull request closed without merging**